packages/server/tests/acceptance/hbarLimiter.spec.ts

```diff
@@ -158,10 +164,69 @@ describe('@hbarlimiter HBAR Limiter Acceptance Tests', function () {
         await deployContract(mediumSizeContract, accounts[0].wallet);
 
         const remainingHbarsAfter = Number(await metrics.get(testConstants.METRICS.REMAINING_HBAR_LIMIT));
-        const expectedCost = 354819247;
+        const expectedCost = 331926551;
         verifyRemainingLimit(expectedCost, remainingHbarsBefore, remainingHbarsAfter);
       });
 
+      it('HBAR limiter is updated within acceptable tolerance range in relation to actual spent amount by the relay operator', async function () {
+        const TOLERANCE = 0.01;
+
+        await new Promise((r) => setTimeout(r, 3000));
```

Comment:
QQ: Why are we waiting for 3 seconds here before we have done anything in the test?

Reply:
This timeout was added to help isolate the metric results from the other tests. I moved this to `beforeAll` for this test suite. Basically it should reduce test flakiness